### ISSUE TITLE
Please add low tom 

### ISSUE THREAD

**Issue by MDJLengers** [montulli/GrooveScribe#29]
I know I can add a low tom by changing the code but it won't play a sound. please add the low tom as default option.

**Comment by montulli** [comment]
Good idea!   I have the support in the code, but I don't have an audio sample for a low 16 inch tom.    All the samples are recorded from Mike Johnston's drum set, so I will need him to set up a low tom and record it with similiar ambiance to the other toms.

**Comment by retorquere** [comment]
> Good idea! I have the support in the code, but I don't have an audio sample for a low 16 inch tom. All the samples are recorded from Mike Johnston's drum set, so I will need him to set up a low tom and record it with similiar ambiance to the other toms.

I'd be happy to donate a oneshot pack from drumdrops; the drumdrops packs are copyright-free according to their [FAQ](https://www.drum-drops.com/pages/faqs). 2Box also has an [extensive set of samples](https://2box-drums.com/sounds/) on their site, I believe they are free to use (see the text at the top of that page), and I have kitbanks that bring together combinations of these samples together as cohesive sets.

**Comment by leocaseiro** [comment]
Hey @montulli , isn't this the same as low tom? 

https://github.com/montulli/GrooveScribe/blob/master/soundfont/NewDrumSamples/MP3/Floor%20Tom.mp3

**Comment by tomweiler** [comment]
Hi,

I found a fork of groovescribe with 3 toms:
https://github.com/BenT-o/GrooveScribe

Many thanks to  Ben Taylor:  https://github.com/BenT-o   !!!!!!

I found  the fork with google and the help of this page:  https://drummersbrainchild.com/

**Comment by jamesiri** [comment]
@montulli any word on this or should we all move on?

**Comment by leocaseiro** [comment]
Can use here https://leocaseiro.github.io/GrooveScribe/

**Comment by montulli** [comment]
I think adding a 3rd tom makes the UI too cluttered for the general use case.  We need to find a way to turn it on and off for advanced users.

**Comment by leocaseiro** [comment]
@montulli Would more spacing (padding) between lines work?

I don't know why would be too cluttered, there're a lot of drum machines that have way more lines and elements than GrooveScribe. 

Another reference is the https://daily-drum.com/drum-notation-editor/editor 

Some other drum machines are: 
https://www.musicca.com/drum-machine and https://drumbit.app/

**Comment by SirJohnDoe** [comment]
Looks great @leocaseiro, thank you. Also thank for the original tool montulli!

Had my 2nd lesson this week and my teacher said nice tool, but there's a Tom missing. So glad I can use the fork. With the extra spacing it's pretty clear for writing things up, and for the actual reading/viewing there's a different view.
